desc/grid.py

```diff
@@ -77,52 +113,52 @@ def _enforce_symmetry(self):
             Nodes on the symmetry line should not be rescaled.
 
         """
-        if self.sym:
-            # indices where theta coordinate is off the symmetry line of theta=0 or pi
-            off_sym_line_idx = np.where(self.nodes[:, 1] % np.pi != 0)[0]
-            __, inverse, off_sym_line_per_rho_surf_count = np.unique(
-                self.nodes[off_sym_line_idx, 0], return_inverse=True, return_counts=True
-            )
-            # indices of nodes to be deleted
-            to_delete_idx = np.where(self.nodes[:, 1] > np.pi)[0]
-            __, to_delete_per_rho_surf_count = np.unique(
-                self.nodes[to_delete_idx, 0], return_counts=True
-            )
-            assert (
-                2 * np.pi not in self.nodes[:, 1]
-                and off_sym_line_per_rho_surf_count.size
-                >= to_delete_per_rho_surf_count.size
+        if not self.sym:
+            return
+        # indices where theta coordinate is off the symmetry line of theta=0 or pi
+        off_sym_line_idx = self.nodes[:, 1] % np.pi != 0
```

Comment:
nothing changed in this block i just switched it to a boolean mask instead of integer because its less code